data/json/items/armor.json

```diff
@@ -12247,14 +12247,14 @@
         "material" : ["cotton", "leather"],
         "volume" : 10,
         "cutting" : 0,
-        "warmth" : 35,
+        "warmth" : 40,
         "phase" : "solid",
         "environmental_protection" : 4,
-        "encumbrance" : 3,
+        "encumbrance" : 2,
         "bashing" : 0,
         "flags" : ["VARSIZE", "WATERPROOF"],
         "coverage" : 95,
-        "material_thickness" : 4
+        "material_thickness" : 5
```

Comment:
>Could perhaps go the other way and instead of buffing the gear, make it avaliable earlier? I could see it being useful in midgame, but hardly in late game.

This. Do this.